include/arch/arm/aarch32/nmi.h

```diff
@@ -1,7 +1,7 @@
 /**
  * @file
  *
- * @brief NMI routines for ARM Cortex M series
+ * @brief NMI routines for ARM
```

Comment:
32-bit

Reply:
Changed to `ARM AArch32` for consistency with all the others